articles/quickstart/backend/java-spring-security5/01-authorization.md

```diff
@@ -0,0 +1,291 @@
+---
+title: Authorization
+name: How to secure your API using Spring Security 5 and Auth0
+description: This tutorial demonstrates how to add authorization to a an API using Spring Security 5.
+budicon: 500
+topics:
+    - quickstart
+    - backend
+    - java
+    - spring-security
+github:
+    path: 01-Authorization-MVC
+contentType: tutorial
+useCase: quickstart
+---
+
+<%= include('../../../_includes/_api_auth_intro') %>
+
+<%= include('../_includes/_api_create_new') %>
```

Comment:
I think preamble should go first, before starting with the directions

Reply:
I agree, good catch. We should update other API Quickstarts to do the same (looks like they all place the preamble later, like I did). 

I updated this PR to move the preamble earlier.